html/ui/locales/bg.json

```diff
@@ -558,6 +558,7 @@
    "no_products_found" : "Няма намерени продукти.",
    "no_products_listed_yet" : "Все още не сте вписали продукти за продажба",
    "no_public_key_warning" : "Предупреждение!: вашата сметка няма публичен ключ! Това означава, че не е защитена като други сметки. Трябва да направите изходящ превод да решите проблема.",
+   "no_public_key_warning_note": "<br><strong>Note:<\/strong> This account does not have a public key! Activate your account <a id='act_lnk' href='#' onclick='BRS.activateAccount();return false;'>here</a>",
```

Comment:
I use this for translation: 

https://electronjs.org/apps/i18n-manager

And embedding logic into translations is kind of weird

Reply:
Removed